### DISCUSSION TITLE
Looking for a developer with experience submitting Etsy 3rd party integrations to help us submit our 3rd party integration

### DISCUSSION THREAD

**Discussion by Mylesdickinson** [etsy/open-api#1028] [Q&A]
Hello All !

We are looking for a developer with experience submitting Etsy 3rd party integrations to help us submit our 3rd party integration.

We have hired an agency to help build out a 3rd party public app for the Shopify App Store that app has some great features that could be beneficial to sellers on Etsy. We would like to submit the app to the Etsy App Store. Though our team has no experience in this field we are looking for a developer that can help us with this task !

For more information feel free to reach out to Myles@haze.tools

Best wishes

**Comment by Gareth064** [comment]
He Myles. I have sent an email with a few questions 👍🏻
